src/models/expenses.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const sequelize = require('../dataBase');
+
+const Expense = sequelize.define('expense', {
+  id: {
+    type: DataTypes.UUID,
+    defaultValue: DataTypes.UUIDV4,
+    primaryKey: true,
+  },
+  userId: {
+    type: DataTypes.UUID,
+    primaryKey: true,
```

Comment:
do you need two primaryKeys? I think it is redundant

Reply:
Done